CefSharp.MinimalExample.Wpf/CefSharp.MinimalExample.Wpf.csproj

```diff
@@ -67,8 +67,10 @@
     <ApplicationManifest>app.manifest</ApplicationManifest>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Microsoft.Expression.Interactions, Version=4.5.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL" />
```

Comment:
They come standard with `VS` these days, far as I'm aware they're not part of `.Net`, though been a while since I actually did any actual `WPF` development so things could easily have changed.

How do other projects handle using them as references? When they're included indirectly as they would be when using `CefSharp` would uses run into problems deploying their app with dlls not being available on non dev machines?